### PR TITLE
[ADD][9.0] stock_picking_digitized_signature

### PR DESCRIPTION
**Pull request by cubells**
Depends on
- [x] https://github.com/OCA/web/pull/666  

New module that allows to add digitized signature to stock pickings and send by mail.

cc @Tecnativa

### CODE REVIEW

**Comment by cubells** [comment]
Move from https://github.com/OCA/stock-logistics-warehouse/pull/312

File: stock_picking_digitized_signature/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    user_id = fields.Many2one(
```
Comment (pedrobaeza) [inline stock_picking_digitized_signature/models/stock.py]: Why don't you add this field to the reports?

Comment (cubells) [inline stock_picking_digitized_signature/models/stock.py]: @pedrobaeza 

added.

**Review by pedrobaeza — APPROVED** [review]
@rafaelbn please review

**Comment by pedrobaeza** [comment]
Travis error unrelated

**Review by rafaelbn — APPROVED** [review]
Tested 👍

**Pull request merged**